### PR TITLE
Add Snapcraft 101 to /training

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Add new section to /training for Snapcraft 101, based on [copydoc](https://docs.google.com/document/d/1gyxy75qXC5Ta08o6Vnm13yK6aeN4qOnsETB2gZ4e0ZY/edit#)

## QA

- Go to /training and compare against [copydoc](https://docs.google.com/document/d/1gyxy75qXC5Ta08o6Vnm13yK6aeN4qOnsETB2gZ4e0ZY/edit#)
- Click 'contact us' prompt and see a custom snapcraft 101 contact us page.
- Fill in the form and see you are directed to a custom snapcraft 101 thank you

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6341

### CODE REVIEW

File: templates/training/index.html [hunk 1]
```diff
@@ -452,7 +452,41 @@ <h2 itemprop="name">Charm Developer</h2>
...
+          <dt class="p-inline-definition-list__title">Duration</dt>
+          <dd class="p-inline-definition-list__item">2 days</dd>
+          <dt class="p-inline-definition-list__title">Cost</dt>
+          <dd class="p-inline-definition-list__item" itemprop="offers" itemscope itemtype="https://schema.org/Offer">USD <span itemprop="priceCurrency" content="USD">$</span><span itemprop="price" content="5000">5,000</span> up to 15 attendees</dd>
+          <dt class="p-inline-definition-list__title">Location</dt>
+          <dd class="p-inline-definition-list__item">Remote</dd>
+          <dt class="p-inline-definition-list__title">Availability</dt>
+          <dd class="p-inline-definition-list__item">Private groups only</dd>
+        </dl>
+        <p><a href="/training/contact-us?product=snapcraft-101" data-form-id="1223" data-lp-id="1973" data-return-url="https://www.ubuntu.com/training/thank-you?product=snapcraft-101" data-lp-url="https://www.ubuntu.com/training/thank-you?product=snapcraft-101">Contact us to book yours&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/training/index.html:483]: Do you need to formid and returnURL if they are defined on the contact page?

**Review by anthonydillon — APPROVED** [review]
Code and QA looks good. One small question in the code.

File: templates/training/index.html (hunk 1 above)
Comment (petesfrench) [reply templates/training/index.html:483]: I had wondered the same thing, it is a pattern that is used on all of the links so I included it. If you think they can be removed, I can remove them from all of the links on the page?

Comment (anthonydillon) [reply templates/training/index.html:483]: If you can, leaving code in a better way then you found it is always a good thing :+1: thanks.

**Pull request merged**